guides/serialization_and_saving.py

```diff
@@ -60,33 +57,8 @@
 import tensorflow as tf
 from tensorflow import keras
 
-"""
-## Whole-model saving & loading
```

Comment:
With this chunk removed the heading nesting doesn't make sense anymore. `### SavedModel` and `### keras H5` are now both under `## Setup`.

Reply:
Updated it, thanks for catching this! :)